buy_together_app/views.py

```diff
@@ -7,3 +9,16 @@ def main_page(request):
 
 def description_page(request):
     return render(request, 'buy_together_app/description.html')
+
+
+def log_in_page(request):
+    if request.method == 'POST':
+        user_name = request.POST.get('username')
+        user_pass = request.POST.get('password')
```

Comment:
do note I asked @thetaxas2468 to add encryption to the passwords, it may require decryption on your side.

Reply:
the built in user do hashing to the password do no need to encrypt and decrypt